INSTALL.md

```diff
@@ -54,189 +54,197 @@ to stop the application, run:
 
-#### Eventmachine
-
-Eventmachine has to be installed with `--with-cppflags=-I/usr/local/opt/openssl/include`. Simply run:
-
-```shell
-bundle config build.eventmachine --with-cppflags=-I/usr/local/opt/openssl/include
-```
-
 #### Config files
 
 Copy example config files from the examples folder:
 
-```shell
+```
 cp config/examples/* config/
 ```
 
-#### Bundle
+#### Bundler
 
-On MacOS 10.15 or newer, first configure the bundle config with:
-```shell
-bundle config --global build.eventmachine --with-cppflags=-I/usr/local/opt/openssl/include
-bundle config --global build.mysql2 "--with-opt-dir=/usr/local/opt/openssl"
-bundle config --local build.github-markdown --with-cflags="-Wno-error=implicit-function-declaration"
-bundle config --local build.thin --with-cflags="-Wno-error=implicit-function-declaration"
+Install [Bundler](https://bundler.io/) to manage all required Ruby gems:
 ```
-
-Run [Bundler](https://bundler.io/) to install all required Ruby gems:
-
-```shell
-bundle install
+gem install bundler
 ```
 
-If the `mysql2` gem installation fails with the error:
-
-```
-ld: library not found for -lssl
+Then configure the bundle config with:
+```sh
+bundle config --global build.eventmachine --with-cppflags="-I$(brew --prefix openssl)/include"
+bundle config --global build.mysql2 --with-opt-dir="$(brew --prefix openssl)"
+bundle config --local build.github-markdown --with-cflags=-Wno-error=implicit-function-declaration
+bundle config --local build.thin --with-cflags=-Wno-error=implicit-function-declaration
 ```
 
-you can fix it setting the flags:
-
-```shell
-bundle config --local build.mysql2 "--with-ldflags=-L/usr/local/opt/openssl/lib --with-cppflags=-I/usr/local/opt/openssl/include"
+And finally install all gems:
+```
+bundle install
 ```
 
-and run `bundle install` again.
+> If the `mysql2` gem installation fails with the error:
+>
+> ```
+> ld: library not found for -lssl
+> ```
+>
+> you can fix it setting the flags:
+>
+> ```sh
+> bundle config --local build.mysql2 --with-ldflags="-L$(brew --prefix openssl)/lib" --with-cppflags="-I$(brew --prefix openssl)/include"
+> ```
+>
+> and run `bundle install` again.
 
-#### Node packages
+#### Yarn
 
-Install [Yarn](https://yarnpkg.com/):
+Install [Yarn](https://yarnpkg.com/) to manage all required Javscript packages:
 
-```bash
+```
 brew install yarn
 ```
 
+And install them:
 
-Run [Yarn](https://www.yarnpkg.com/) to install all the required dependencies:
-
-```bash
+```
 yarn install
 ```
 
 ## Manual setup on Fedora (34)
 
 ### Prerequisites
 
-#### Ruby version
-
-The project supports **2.6.x**.
+#### Package and runtime version management
 
-Verify you have a proper version by running on your terminal:
+To manage the required runtimes, such as Ruby and Node.js, we recommend using [asdf](https://asdf-vm.com/guide/getting-started.html#global). Install it directly from Github:
+```
+dnf install curl git
+git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.10.2
+```
```

Comment:
Instead of this, I would say "Follor the instructions at [asdf sire](https://asdf-vm.com/guide/getting-started.html#_2-download-asdf) to install it". Since it needs to add some lines to `.bashrc` which doesn't appear in this guide, and maybe they change that in future releases. So better just link there.

Reply:
What about:
>To manage the required runtimes, such as Ruby and Node.js, we recommend using [asdf](https://asdf-vm.com/guide/introduction.html). See its [getting started](https://asdf-vm.com/guide/getting-started.html) guide in order to get specific instructions for your SHELL and installation method.

Reply:
That's perfect